# Disables setting the Zone Identifier on downloaded files. It is a minor annoyance to remove every time.

--- a/content/browser/safe_util_win.cc
+++ b/content/browser/safe_util_win.cc
@@ -54,40 +54,7 @@ bool SetInternetZoneIdentifierDirectly(const base::FilePath& full_path) {
 HRESULT AVScanFile(const base::FilePath& full_path,
                    const std::string& source_url,
                    const GUID& client_guid) {
-  base::win::ScopedComPtr<IAttachmentExecute> attachment_services;
-  HRESULT hr = attachment_services.CreateInstance(CLSID_AttachmentServices);
-
-  if (FAILED(hr)) {
-    // The thread must have COM initialized.
-    DCHECK_NE(CO_E_NOTINITIALIZED, hr);
-
-    // We don't have Attachment Execution Services, it must be a pre-XP.SP2
-    // Windows installation, or the thread does not have COM initialized. Try to
-    // set the zone information directly. Failure is not considered an error.
-    SetInternetZoneIdentifierDirectly(full_path);
-    return hr;
-  }
-
-  if (!IsEqualGUID(client_guid, GUID_NULL)) {
-    hr = attachment_services->SetClientGuid(client_guid);
-    if (FAILED(hr))
-      return hr;
-  }
-
-  hr = attachment_services->SetLocalPath(full_path.value().c_str());
-  if (FAILED(hr))
-    return hr;
-
-  // Note: SetSource looks like it needs to be called, even if empty.
-  // Docs say it is optional, but it appears not calling it at all sets
-  // a zone that is too restrictive.
-  hr = attachment_services->SetSource(base::UTF8ToWide(source_url).c_str());
-  if (FAILED(hr))
-    return hr;
-
-  // A failure in the Save() call below could result in the downloaded file
-  // being deleted.
-  return attachment_services->Save();
+  return S_OK;
 }
 
 }  // namespace content
